### PR TITLE
Add a custom keymap for Torn keyboard

### PR DESCRIPTION
**Pull request by Cy6erBr4in**
## Description

This PR introduces an additional keymap for Torn keyboard. This keymap was inspired by default layout of Kinesis Advantage (all modifier keys as well as layer keys are duplicated on both halves of the keyboard).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by rtitmuss — APPROVED** [review]
lgtm

File: keyboards/torn/keymaps/kinesish/config.h [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#define TAPPING_TERM 200
```
Comment (drashna) [inline keyboards/torn/keymaps/kinesish/config.h:1]: This needs a license header, and an include guard (`#pragma once`)

Comment (Cy6erBr4in) [reply keyboards/torn/keymaps/kinesish/config.h:19]: done and done!

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/torn/keymaps/kinesish/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# A Custom Keymap for [Torn](https://github.com/rtitmuss/torn)
```
Comment (fauxpark) [inline keyboards/torn/keymaps/kinesish/README.md:1]: Oops, one thing actually - could you rename this file to lowercase?

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/torn/keymaps/kinesish/README.md (hunk 2 above)
Comment (Cy6erBr4in) [reply keyboards/torn/keymaps/kinesish/README.md:1]: sure thing!

Comment (Cy6erBr4in) [reply keyboards/torn/keymaps/kinesish/README.md]: and done.

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**